app/guid-file/styles.scss

```diff
@@ -78,7 +79,47 @@
 .RightContainer {
     margin: 0 20px;
     
+    h2:nth-of-type(2) {
+        margin-bottom: 1vh;
+    }
+
+    h3 {
+        font-weight: 800;
+        font-size: 15px;
+        margin-top: 4px;
+    }
+
     h2 {
-        margin-bottom: 20px;
+        padding-bottom: 1vh;
+    }
+
+    div {
+        margin-bottom: 2vh;
+    }
+
+    div:nth-of-type(1) {
```

Comment:
For these nth-of-type items, would it make more sense to just give them classes instead? Or is there no way of separating out which item would belong to a particular class other than by position on the page?

Reply:
Ok. I have gone through and updated nested CSS selectors to use local class selectors within the handlebars file. One for the infinite list remains, where spans are appended to a list of link elements should there not be a OSF profile associated with the user. For this to stay within the project scope, I have chosen to keep this selector until we want to address the ContributorList's InfiniteList element.